app.py

```diff
@@ -419,19 +421,16 @@ def handle_search(keyword):
 
     results = []
     for schedule in get_schedule_data().values():
-        test_keyword = get_name(schedule)
+        test_keyword = get_first_name(schedule) + " " + schedule["lastname"]
         if keyword.lower() in test_keyword.lower():
             results.append({"name": test_keyword, "username": schedule["username"]})
             if len(results) >= 5:  # We only display five results
                 break
     return json.dumps(results)
 
 
-def get_name(schedule):
```

Comment:
I think it would be good to keep `get_name` (or maybe call it `get_full_name`) so you can still use it on line 424